src/script/cpp_api/s_env.cpp

```diff
@@ -105,7 +105,6 @@ void ScriptApiEnv::initializeEnvironment(ServerEnvironment *env)
 			lua_pushnil(L);
 			while (lua_next(L, table)) {
 				// key at index -2 and value at index -1
-				luaL_checktype(L, -1, LUA_TSTRING);
```

Comment:
Couldn't we call this function though `lua_pcall` after pushing the C function to the stack?

Reply:
Also, if the error were caught here, a backtrace could not be provided.